zboss/src/CMakeLists.txt

```diff
@@ -13,6 +13,13 @@ zephyr_library_include_directories(
     include
   )
 
+# Prevent the -Waddress-of-packed-member warning. We appear to be
+# taking addresses of struct fields in a manner that is not
+# safe. Until this is resolved upstream, we temporarily remove this
+# warning.
+zephyr_library_compile_options(-Wno-address-of-packed-member)
+
+
```

Comment:
minor nit: why the extra empty line ?
```suggestion
```

Reply:
Accidental.

Would prefer to not do any git paperwork for this, but will fix if I have to touch the code any way.